causal_testing/testing/estimators.py

```diff
@@ -495,7 +495,7 @@ def add_modelling_assumptions(self):
             (iii) Instrument and outcome do not share causes
         """
 
-    def estimate_coefficient(self, df):
+    def estimate_coefficient_aux(self, df):
```

Comment:
What does aux stand for here? It might not be obvious to users the difference between the `estimate_coefficient` and `estimate_coefficent_aux` methods

Reply:
Auxiliary. I think this is semi-standard practice in functional programming, but is probably not "pythonic". I renamed the method to "estimate_iv_coefficient"